test/powershell/Modules/Microsoft.PowerShell.Utility/Tee-Object.Tests.ps1

```diff
@@ -4,21 +4,35 @@ Describe "Tee-Object" -Tags "CI" {
 
     Context "Validate Tee-Object is correctly forking output" {
 
-	$testfile = Join-Path -Path (Join-Path -Path $PSScriptRoot -ChildPath assets) -ChildPath testfile.txt
+        BeforeAll {
+            $testfile = Join-Path -Path (Join-Path -Path $TestDrive -ChildPath assets) -ChildPath testfile.txt
+            $testvalue = [char](244)
+            if ($IsWindows) {
+                $expectedBytes = 244,13,10 -join "-"
+            } else {
+                $expectedBytes = 244,10 -join "-"
+            }
+            Remove-Item -Path $teefile -ErrorAction SilentlyContinue -Force
```

Comment:
Please move to BeforeEach block.

Reply:
Sorry.. messed up BeforeAll and BeforeEach...

Reply:
It should seem fix CIs.
```suggestion
            Remove-Item -Path $testfile -ErrorAction SilentlyContinue -Force
```